### ISSUE TITLE
An onEventSubscribed/onEventUnsubscribed

### ISSUE THREAD

**Issue by osiloke** [desertbit/bitmonster#1]
There is no confirmation that an event has been properly subscribed to. The client has no way of telling if an event is subscribed and can be waiting naively for no data. There should at least be an error callback.
What i'm doing now to mimic onEventSubscribed, is to subscribe to an event and then call a method which checks that the event subscription was successful

**Comment by m4ng0squ4sh** [comment]
For what kind of case do you need a success callback? Would like to understand that use-case. I agree that an error callback could me useful. I'll integrate that...

**Comment by osiloke** [comment]
Yeah, I meant only an error callback, we are on the same page. That will be awesome.

**Comment by m4ng0squ4sh** [comment]
I added this to the TODO. I'll implement it, as soon as I'll implement the fileserver for BitMonster ;)
